todo-backend-springboot/src/main/kotlin/epo/todo/backend/exception/RestExceptionHandler.kt

```diff
@@ -0,0 +1,23 @@
+package epo.todo.backend.exception
+
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.ControllerAdvice
+import org.springframework.web.bind.annotation.ExceptionHandler
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler
+
+@ControllerAdvice
+class RestExceptionHandler : ResponseEntityExceptionHandler() {
```

Comment:
needs a fallback case returning 500.
also needs stack trace logging. add this when solving logging issue #9